libbeat/docs/shared-template-load.asciidoc

```diff
@@ -37,42 +37,60 @@ For more information, see:
 
 By default, {beatname_uc} automatically loads the recommended template file, +fields.yml+,
 if Elasticsearch output is enabled. You can configure {beatname_lc} to load a different template
-by adjusting the `template.name` and `template.fields` options in +{beatname_lc}.yml+ file:
+by adjusting the `setup.template.name` and `setup.template.fields` options in
++{beatname_lc}.yml+ file:
 
 ["source","yaml",subs="attributes,callouts"]
 ----------------------------------------------------------------------
-output.elasticsearch:
-  hosts: ["localhost:9200"]
-  template.name: "{beatname_lc}"
-  template.fields: "fields.yml"
-  template.overwrite: false
+setup.template.name: "{beatname_lc}"
+setup.template.fields: "path/to/fields.yml"
+setup.template.overwrite: false
 ----------------------------------------------------------------------
 
 By default, if a template already exists in the index, it is not overwritten. To overwrite an existing
-template, set `template.overwrite: true` in the configuration file.
+template, set `setup.template.overwrite: true` in the configuration file.
 
-To disable automatic template loading, comment out the template part under the Elasticsearch output.
+To disable automatic template loading, set `setup.template.enabled: false`.
 
 The options for auto loading the template are not supported if you are using the
 Logstash output.
 
 [[load-template-manually]]
 ==== Load the template manually
 
-If you disable automatic template loading, you need to run the following command to load the template:
+If you disable automatic template loading, you can run the `setup` command to
+load the template.
+
+["source","sh",subs="attributes"]
+----
+./{beatname_lc} setup --template
+----
+
+ifeval::["{requires-sudo}"=="yes"]
+
+If you changed ownership of the config file to root, you'll need preface this
+command with `sudo`.
+
+endif::[]
+
+//REVIEWERS: I'm keeping separate steps for the loading the index template and the dashboards since it seems like the index template loading happens automatically, but the dashboard loading must be specified. It might be confusing to describe this in one step. Plus having separate steps helps users understand that two very different operations are taking place (the loading of the index template into ES and the loading of dashboards into Kibana)
+
+//TODO: I think this needs to be deleted. Commenting it out for now just in case. Also need to remove conditional flag
```

Comment:
Can someone confirm that there is no reason to keep the content commented out here?